CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+CHANGES IN VERSION 1.21.0 (FROM 1.20.2)
+===================================
+
+Appium 1.21.0 is a minor release
+
+### General
+
+* Minimum NodeJS version moved to 12
+
+### Android General
+
```

Comment:
Switching to web views with same package name but different process names has been fixed: https://github.com/appium/appium-android-driver/pull/687

Reply:
Sending of Google analytics data by instruments has been disabled